flask_appbuilder/security/sqla/apis/user/api.py

```diff
@@ -0,0 +1,203 @@
+from datetime import datetime
+
+from flask import current_app
+from flask import g, request
+from flask_appbuilder import ModelRestApi
+from flask_appbuilder.api import expose, safe
+from flask_appbuilder.const import API_RESULT_RES_KEY
+from flask_appbuilder.models.sqla.interface import SQLAInterface
+from flask_appbuilder.security.decorators import permission_name, protect
+from flask_appbuilder.security.sqla.models import Role, User
+from marshmallow import ValidationError
+from sqlalchemy.exc import IntegrityError
+from werkzeug.security import generate_password_hash
+
+from .schema import UserPostSchema, UserPutSchema
+
+
+class UserApi(ModelRestApi):
+    resource_name = "user"
+    class_permission_name = "User"
+    datamodel = SQLAInterface(User)
+    allow_browser_login = True
+
+    list_columns = [
+        "id",
+        "roles.id",
+        "roles.name",
+        "first_name",
+        "last_name",
+        "username",
+        "active",
+        "email",
+        "last_login",
+        "login_count",
+        "fail_login_count",
+        "created_on",
+        "changed_on",
+        "created_by.id",
+        "changed_by.id",
+        "created.id",
```

Comment:
consider removing the `created` and `changed` backref's for performance reasons. Not relevant info IMO

Reply:
I thought by default all info possible and let user specify only they columns they need.

Reply:
I think it's dangerous, on dbs with lot's of users

Reply:
agree, removed already.